src/modules/events/events.service.ts

```diff
@@ -64,4 +72,52 @@ export class EventsService {
       where: { id },
     });
   }
+
+  async inviteUserToEvent(
+    eventId: number,
+    inviteUserDto: InviteUserDto,
+  ): Promise<void> {
+    const event = await this.prismaService.event.findUnique({
+      where: { id: eventId },
+    });
+
+    if (!event) {
+      throw new NotFoundException('Event not found');
+    }
+
+    const { userIds } = inviteUserDto;
+    const data = userIds.map((userId) => {
+      return {
+        eventId: event.id,
+        userId,
+        status: UsersOnEventStatus.PENDING,
+        role: EventUserRole.PARTICIPANT,
+      };
+    });
+
+    await this.prismaService.usersOnEvents.createMany({
+      data,
+    });
+  }
+
+  async confirmInvitation(
+    eventId: number,
+    userId: number,
+    confirmInvitationDto: ConfirmInvitationDto,
+  ): Promise<UsersOnEvents> {
+    const { status, reason } = confirmInvitationDto;
+
+    return this.prismaService.usersOnEvents.update({
+      where: {
+        userId_eventId: {
+          eventId,
+          userId,
+        },
+      },
+      data: {
+        status,
+        reason,
```

Comment:
- Chỗ này không phải lúc nào `data update` cũng có `status` với `reason` (chỉ khi cancel invitation event mới có reason) nên chỗ này em tránh hard code `data: { reason, status }` => nó truyền lên gì thì update cái đấy thôi em.
```suggestion
    return this.prismaService.usersOnEvents.update({
      where: {
        userId_eventId: {
          eventId,
          userId,
        },
      },
      data: confirmInvitationDto
```